cdap-docs/developer-manual/source/getting-started/start-stop-cdap.rst

```diff
@@ -45,3 +45,13 @@ applications and interact with CDAP.
 Note that in the case of the Docker container running inside a Virtual Machine (as on
 either Mac OS X or Microsoft Windows), you will need to substitute the Docker VM's IP
 address for ``localhost`` in the web browser address bar.
+
+
+Customize JVM OPTS For Sandbox on Linux
+---------------------------------------
+On Linux, you can customize the jvm options for OutOfMemory error and whether to do a heapdump if an OutOfMemory error
+happens. By default, CDAP process will be force killed if an OutOfMemory error happens and there will be no heapdump.
+You can configure these settings by editing the ``cdap-env.sh`` file under your ``conf`` directory. To change
+the behavior, change the value of ``KILL_ON_OOM_OPTS`` in ``cdap-env.sh``. To enable the heapdump, uncomment the line of
```

Comment:
"To enable the heap dump, ensure ``HEAPDUMP_ON_OOM`` is set to ``true`` in ``cdap-env.sh``" . (this is just to make the docs more resilient if we ever change that line in cdap-env.sh)